test-gen-plugin/src/main/java/com/dylibso/chicory/maven/JavaTestGen.java

```diff
@@ -200,22 +205,29 @@ public CompilationUnit generate(
                         }
                     }
                     break;
+                case REGISTER:
+                    // should be irrelevant
+                    break;
                 case ASSERT_MALFORMED:
```

Comment:
What does this mean exactly? Should we log a message here? No strong opinion just curious.

Reply:
We are not doing anything for the `register` and is the same in wazero.
We still pass the tests 🙂 I'm becoming more confident that this is not useful at all in the way we are running the tests.